portfolio/src/main/webapp/style.css

```diff
@@ -1,9 +1,93 @@
-#content {
+* { 
+    font-family: 'Didact Gothic';
+    color: whitesmoke;
+}
+
+body {
+background-image: linear-gradient(#434343,#000000);
+}
+
+.nav-items {
+  list-style-type: none;
+  margin: 0;
+  padding: 0;
+  overflow: hidden;
+  background-color: #333;
+}
+
+.nav-item {
+    float: left;
+  display: block;
+  color: white;
+  text-align: center;
+  padding: 14px 16px;
+  text-decoration: none;
+}
+
+/* Change the link color to #111 (black) on hover */
+.nav-item:hover {
+  background-color: rgb(24, 57, 163);
+}
+
+.parallax {
+    background-image: url("../images/fiberOptic.jpg");
+    height: 500px;
+    background-attachment: fixed;
+    background-position: center;
+    background-repeat: no-repeat;
+    background-size: cover;
+    filter: blur(4px);
+    -webkit-filter: blur(4px); 
+}
+
+.centered {
+  position: absolute;
+  font-size: 40px;
+  top: 50%;
+  left: 50%;
+  transform: translate(-50%, -50%);
+}
+
+#profile-pic {
+    border-radius: 50%;
+    width: 30%;
+    height: 0%;
+    transition: height 0.5s;
+    transition: width 0.5s;
+}
+
+#profile-pic:hover {
+    height: 35%;
+    width: 35%;
+}
+
+.content {    
   margin-left: auto;
   margin-right: auto;
   width: 650px;
 }
 
-#greeting-container {
-  margin-top: 20px;
+.languages {
+    height: 10%;
+    width: 10%;
+    transition-duration: 0.3s;
 }
+
+.languages:hover {
+    transform: perspective(1px) translateZ(5%);
+}
+
+.category-image {
+    justify-content: center;
+    width : 90%;
+    height: 200px;
+    padding: 1%;
+    transition: ease-in 1s ease-out 2s;
+    filter: blur(4px);
+    -webkit-filter: blur(4px); 
+}
+
+.category-image:hover {
+    filter: blur(0);
+    -webkit-filter: blur(0);
+}
```

Comment:
nit: Add a new line at the end of this file. Notice that github puts a red mark here if there is no new line. This is just a small format request that github enforces.